requirements.txt

```diff
@@ -4,7 +4,6 @@
 #
 #    pip-compile --no-index --output-file requirements.txt requirements.in
 #
-colorama==0.3.9           # via structlog
```

Comment:
Could you add `colorama` as a direct dependency? It improves log readability, and it's a pretty simple dependency.

Reply:
Added as suggested.

Reply:
Thanks!